index.html

```diff
@@ -0,0 +1,188 @@
+      <div class="main-program">
+        <h3>Main Program</h3>
+        <hr>
+      </div>
+
+      <div class="wesing_programs">
+        <div class="program">
+          <i class="fas fa-users fa-2x"></i>
+          <h4>Counselling</h4>
+          <p>Get answers to various issues of life from our experienced and spiritually inclined counsellors during our
+            two-hour counselling session</p>
+        </div>
+        <div class="program program_2">
+          <i class="fas fa-pray fa-2x"></i>
+          <h4>Intercession</h4>
+          <p>One hour intercessory prayer for the nations of Africa, governments, global economies, families and
+            individuals. Write your prayer requests.</p>
+        </div>
+        <div class="program program_2">
+          <i class="fas fa-praying-hands fa-2x"></i>
+          <h4>WARship</h4>
+          <p>The Father seeks those who will worship him in spirit and in truth. Get ready to pour your heart out during
+            our four-hour WARship session.</p>
+        </div>
+        <div class="program">
+          <i class="fas fa-volume-up fa-2x"></i>
+          <h4>PraiseParty</h4>
+          <p>Are you ready to danceand sing praises to the Most High? Learn all the amazing dance moves to think you can
+            pull off because we'll be dancing for four hours.</p>
+        </div>
+
+      </div>
+
+      <div class="program_button">
+        <a href="#">See the whole program</a>
+        <button type="submit">Book A Counselling Session</button>
+      </div>
+
+    </section>
+
+    <!-- Invited Guests Section -->
+    <section id="wesing_guests">
+      <div class="main-program">
+        <h3>Guests</h3>
+        <hr>
+      </div>
+
+      <!--Content Of This Section Will Be Displayed Dynamically On Page Load using JavaScript. See guests.js for code.  -->
+      <div class="guest_list"></div>
+
+      <div class="guest_button" id="show_guests">
+        <button type="submit" >More <i class="fas fa-chevron-down"></i></button>
+      </div>
+    </section>
+
+    <!-- Partners -->
+    <section id="wesing_partners">
+      <div class="main-program">
+        <h3>Partners</h3>
+        <hr>
+      </div>
+
+      <div class="partners">
+        <img src="./images/goog.png" alt="Mozilla">
+        <img src="./images/mozil.png" alt="Google">
+        <img src="./images/naver.png" alt="Naver">
+        <img src="./images/daum.png" alt="Daum">
+        <img src="./images/airbnb.png" alt="Airbnb">
+      </div>
+    </section>
+  </main>
+
+  <!-- Footer -->
+  <footer>
+    <div class="footer_copy">
+      <img src="./images/wesing-logo.png" alt="Footer Logo">
+      <aside>
+        <h6>WeSingAfrica <br> Gospel Concert</h6>
+      </aside>
+    </div>
+
+    <div class="final-parag">
+      <p>2021 WeSingAfrica Nigeria</p>
+      <p>Some Rights Reserved</p>
+    </div>
+
+    <div class="go-up">
+      <a href="#"><i class="fas fa-arrow-up"></i></a>
+    </div>
+  </footer>
```

Comment:
- [ ] For the footer is important to consider that the desktop size could have a dark background in place of a white one (the white is excellent on mobile, but talking about the desktop size, please, change the color to dark). 🐱 
- Hint: I observe the about footer is dark, so, just use the same on the home page (The footer and header should be the same).
![image](https://user-images.githubusercontent.com/20563252/128189919-9e88a8d8-4e78-4fe1-b028-8b7fe99afacc.png)

Reply:
Thank you for the feedback, Victor.
The template has different colors for the footer on the home and about sections. This is because the partners are only present in the desktop version of the home page and it already carries the same background color as the about page footer. Changing the background color of the home page footer on desktop to the same one as the about page footer will make the 'partners and footer' sections on the home page look like they're in the same section.

Cheers 😀

Reply:
Hello there, I share with you some of the requirements for complete the project. 😄 
![image](https://user-images.githubusercontent.com/20563252/128219223-1b029743-787b-43a2-8994-c8abc5f5b1e9.png)
![image](https://user-images.githubusercontent.com/20563252/128219308-eb59d129-89bc-4cad-863f-9fac2a2caa7e.png)
I hope this could help.